src/renderers/shared/reconciler/ReactUpdateQueue.js

```diff
@@ -138,14 +145,6 @@ var ReactUpdateQueue = {
   },
 
   enqueueCallbackInternal: function(internalInstance, callback) {
-    invariant(
```

Comment:
I removed this one because it’s not enough for `ReactDOM.render()` code path. `ReactMount` calls the callback synchronously on the initial render so we would fail to print the error if we only checked here.

Currently `ReactMount` is the only thing using `enqueueCallbackInternal`. I could put an additional `validateCallback` call here too but then it would run twice on every initial render.